Lib/test/test_strptime.py

```diff
@@ -292,6 +292,10 @@ def test_hour(self):
         self.assertTrue(strp_output[3] == self.time_tuple[3],
                         "testing of '%%I %%p' directive failed; '%s' -> %s != %s" %
                          (strf_output, strp_output[3], self.time_tuple[3]))
+        am_date = (1999, 3, 17, 10, 44, 55, 2, 76, 0)
+        pm_date = (1999, 3, 17, 22, 44, 55, 2, 76, 0)
+        self.assertEqual(_strptime._strptime_time(time.strftime("%I %p", am_date), "%I %p")[3], 10)
```

Comment:
I'm not crazy about testing implementation details like `_strptime`, though I realize that that's what this `test_strptime` file does.

Personally, I think it might be worth migrating more of this stuff into `datetimetester.py` and just testing with `datetime.strftime`/`datetime.strptime` instead.